porcupine/plugins/rightclick_menu_options.py

```diff
@@ -0,0 +1,6 @@
+from porcupine.plugins import rightclick_menu, text_google_search
+
+def setup()->None:
+    rightclick_menu.add_filetab_command("Search Google for text", text_google_search.google_search)
```

Comment:
Put this to the `text_google_search()` plugin. The idea is that each plugin adds its own thing into the right-click menu, similarly to how the menubar at top works.